src/internal/item-container/index.tsx

```diff
@@ -196,6 +196,18 @@ function ItemContainerComponent(
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [item.id, transitionInteractionType, transitionItemId]);
 
+  useEffect(() => {
+    if (transitionInteractionType === "keyboard" && transitionItemId === item.id) {
+      const onPointerDown = () => draggableApi.submitTransition();
+      window.addEventListener("pointerdown", onPointerDown, true);
+      return () => {
+        window.removeEventListener("pointerdown", onPointerDown, true);
+      };
+    }
+    // draggableApi is not expected to change
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [item.id, transitionInteractionType, transitionItemId]);
```

Comment:
This is almost never as easy as "just add useMemo", because that object depends on the other values which change more often than expected.

You are right there are opportunities to revisit this bit (probably via creating this API only once using ref), but not in the scope for my current bug fixes